db/migrate/20160127210624_convert_configurations_to_settings_changes.rb

```diff
@@ -18,15 +18,30 @@ def up
   private
 
   def full_to_deltas(full_config)
-    deltas = Vmdb::Settings::HashDiffer.changes(TEMPLATES[full_config.typ], full_config.settings.deep_symbolize_keys)
+    config      = full_config.settings.deep_symbolize_keys
+    config_type = full_config.typ
+
+    adjust_config!(config_type, config)
```

Comment:
Are we expecting more types that broker notify to be adjusted?  If not, perhaps this method should be called just on broker notify types and be something like `convert_broker_notify_keys_to_array`